.changeset/young-flies-camp.md

```diff
@@ -0,0 +1,5 @@
+---
+'@sveltejs/kit': patch
+---
+
+allow /favicon.ico to be a valid route
```

Comment:
Would it be worth it to make it `[breaking]` with an indication that requests are no longer suppressed? `handle` implementations might be surprised.

```suggestion
[breaking] stop suppressing `/favicon.ico` requests, handle them as a valid route
```